asv_bench/benchmarks/groupby.py

```diff
@@ -361,16 +361,15 @@ def time_category_size(self):
 class FillNA:
     def setup(self):
         N = 1500
-        self.df = DataFrame({
-                    'group' : [1] * N + [2] * N,
-                    'value' : [np.nan, 1.0] * N
-                }).set_index('group')
-    
+        self.df = DataFrame(
+            {"group": [1] * N + [2] * N, "value": [np.nan, 1.0] * N}
+        ).set_index("group")
+
     def time_df_ffill(self):
-        self.df.groupby('group').fillna(method='ffill')
-    
+        self.df.groupby("group").fillna(method="ffill")
```

Comment:
I'm not sure if ASV is usually added for regressions only, but at this point adding `self.df.groupby("group").ffill()` and other methods as `self.df.groupby("group").fillna(method="bfill")` would make sense.

Reply:
yes it is fine to add here as well

Reply:
Done